### PR TITLE
Improved Select All action for diagram elements. Closes #2108

### LINKED ISSUE #2108: Select all action doesn't select all diagram elements.
The select all action provided by GEF only selects top level diagram elements. This is problematic because it makes it difficult to select all elements to adjust contents.

We need to override the Select All implementation with an action that will select all elements similar to how Select All Connections works. This will bring the behavior in line with other common tools.

### PR DESCRIPTION
**Pull request by philip-alldredge**

### CODE REVIEW

**Comment by philip-alldredge** [comment]
@RyanMcilnay please take a look at this.

File: ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/actions/SelectAllAction.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	public SelectAllAction(final AgeDiagramEditor editor) {
+		super(editor);
+		this.editor = Objects.requireNonNull(editor);
+		setId(ActionFactory.SELECT_ALL.getId());
+	}
+
+	@Override
+	public void run() {
+		final GraphitiAgeDiagram gad = editor.getGraphitiAgeDiagram();
+		final PictogramElement[] connectionPictogramElements = editor.getAgeDiagram().getAllDiagramNodes()
```
Comment (RyanMcilnay) [inline ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/actions/SelectAllAction.java]: Rename connectionPictogramElements

File: ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/actions/SelectAllAction.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				.filter(dn -> dn instanceof DiagramElement).map(DiagramElement.class::cast)
```
Comment (RyanMcilnay) [inline ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/actions/SelectAllAction.java]: dn -> dn instanceof DiagramElement could be DiagramElement.class::isInstance (more consistent with DiagramElement.class::cast).  But it is up to you if you want to change it.

**Comment by philip-alldredge** [comment]
@RyanMcilnay your comments didn't show up on github(did you submit your review?). However, I think I've addressed them. Please review.

**Comment by RyanMcilnay** [comment]
Sorry about that, I did not submit the review.  I had one other comment, see above.

**Comment by philip-alldredge** [comment]
@RyanMcilnay fixed. Should be ready now.

**Pull request merged**